optimum/exporters/onnx/model_configs.py

```diff
@@ -632,7 +632,32 @@ def generate_dummy_inputs(self, framework: str = "pt", **kwargs):
         return dummy_inputs
 
 
-class VaeOnnxConfig(ViTOnnxConfig):
+class VaeEncoderOnnxConfig(ViTOnnxConfig):
+    ATOL_FOR_VALIDATION = 1e-2
```

Comment:
It seems huge no?

Reply:
This is due to https://github.com/huggingface/diffusers/blob/v0.11.1/src/diffusers/models/vae.py#L357, currently I've found that a tolerance of `1e-2` was respected for common models